build_tools/ci/cpu_comparison/run.py

```diff
@@ -1122,6 +1182,68 @@ def __init__(self):
                 )
             )
 
+        self.register(
```

Comment:
Add a comment above to show what this test is for.

Reply:
I don't know what this test is for, there was no comment in run_matmul_test.sh